tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -359,6 +359,20 @@
     well_max_flow_rate=Scalar(1.0, "sm3/d"),
     well_max_flow_rate_curve=Curve(Array([1.0, 1.0], "sm3/d"), Array([0, 1], "s")),
 )
+FORCHHEIMER_IPR_DESCRIPTION = case_description.ForchheimerIPRDescription(
```

Comment:
I suggest adding two usages of Forchheimer IPR:
```python
FORCHHEIMER_IPR_DESCRIPTION = case_description.ForchheimerIPRDescription(
    well_index_phase=constants.WellIndexPhaseType.Oil,
    min_pressure_difference=Scalar(0.0, "bar"),
    calculate_coeff_option=constants.ForchheimerCoefficientsOption.ReservoirParameters,
    gas_viscosity=Scalar("dynamic viscosity", 0.0, "Pa.s"),
    gas_z_factor=Scalar("dimensionless", 0.0, "-"),
    reservoir_permeability=Scalar("permeability rock", 0.0, "m2"),
    drainage_radius=Scalar("length", 0.0, "m"),
    well_radius=Scalar("length", 0.0, "m"),
    well_skin_factor=Scalar("dimensionless", 0.0, "-"),
    non_darcy_parameter=Scalar("nonDarcy flow coefficient", 0.0, "Pa.s/m6"),
)
FORCHHEIMER_IPR_COEFF_DESCRIPTION = case_description.ForchheimerIPRDescription(
    well_index_phase=constants.WellIndexPhaseType.Oil,
    min_pressure_difference=Scalar(0.0, "bar"),
    calculate_coeff_option=constants.ForchheimerCoefficientsOption.FlowCoefficients,
    B_coeff=Scalar(1.0, "Pa2.s/scm"),
    C_coeff=Scalar(1.0, "Pa2.s2/scm2"),
)
```
With that, we would have both ways of Forchheimer IPR configuration. What do you think?

Reply:
This is a good idea, but the case description `FORCHHEIMER_IPR_COEFF_DESCRIPTION` will not be tested. If you look at [test_alfacase_to_case.py](https://github.com/ESSS/alfasim-sdk/pull/313/files#diff-039440478d6ecc71845eb828ddc2e38fc2284e7c98d04b8a9162e0128fb7cdc9bR277-R281) you will find a dictionary where the key is `ForchheimerIPRDescription`, which must match the name of the class description. If I want to test `FORCHHEIMER_IPR_DESCRIPTION` and `FORCHHEIMER_IPR_COEFF_DESCRIPTION` a class is needed for each option, which goes against the initial idea of ​​having a single class with all the data.

Reply:
Hmm, I understand. Then, I suggest adding only the model data that calc would use (One of two descriptions that I suggested before).  Is it possible? Or the test would check all possible description parameters?

Reply:
It is done.  Since we are assuming the `ReservoirParameters` as default option, I removed the `B_coeff` and `C_coeff`. Removing these arguments makes them take on their own default value, so I think we're good.